packages/theme/src/cli/commands/theme/dev.ts

```diff
@@ -158,7 +157,7 @@ You can run this command only in a directory that matches the [default Shopify t
     await dev({
       adminSession,
       directory: flags.path,
-      store,
+      store: flags.store,
```

Comment:
To make sure: we can skip using `ensureThemeStore` here now because `ThemeCommand` will run `createSession` which does it instead?